client/finality-grandpa/rpc/src/lib.rs

```diff
@@ -313,6 +313,8 @@ mod tests {
 	}
 
 	#[tokio::test]
+	// Will be removed in commit https://github.com/paritytech/substrate/commit/abb80e54aa260514a4f12712a9ff8ef944e3b4ec
+	#[ignore]
```

Comment:
Could you please add the version where it will be removed?

Reply:
Version added here and in 2nd the same test.